policyengine_canada/parameters/gov/cra/tax/income/credits/gst_credit/reduction_multiplier.yaml

```
@@ -0,0 +1,12 @@
+description: GST credit reduction multiplier
+values:
+  2022-07-01:
```

Can then remove the value. Similarly in other files - we only need the separate `value` attribute if we're providing other metadata for a particular value, like a reference.
```suggestion
  2022-07-01: 0.05
```